src/scss/_main.scss

```diff
@@ -1092,7 +1092,7 @@ details.build-toggle {
 .logs .line {
   display: flex;
   align-items: flex-start;
-  margin: 0 0.5em;
+  margin: 0 0.5em 0 0;
```

Comment:
fixing padding hover bug when introducing the border-left from warning annotations